cwl/src/main/scala/cwl/Workflow.scala

```diff
@@ -50,8 +50,16 @@ case class Workflow private(
   
   lazy val womFqn: Option[wom.graph.FullyQualifiedName] = {
     explicitWorkflowName.value map { workflowName =>
-      parentWorkflowStep.map(_.womFqn.combine(workflowName))
-        .getOrElse(wom.graph.FullyQualifiedName(workflowName))
+      // Sometimes the workflow name is of the form "parentWorkflowStepName/UUID"
+      // In that case, we don't want the fqn to look like "parentWorkflowStepName.parentWorkflowStepName/UUID"
+      // To avoid that, strip the parentWorkflowStepName from the workflowName
```

Comment:
I could maybe let two lines go, but three is too far! 😉 Please put the multiline comment block in a `/* */`